ci/install.sh

```diff
@@ -1,9 +1,12 @@
+#!/bin/bash -eux
```

Comment:
I'd leave in the `e` & `u` for error handling. Let me know if you want the `x` removed which just echos the commands. Seems pretty useful.